CMakeLists.txt

```diff
@@ -1,11 +1,15 @@
-cmake_minimum_required(VERSION 3.12)
-project(funi VERSION 0.0.0 LANGUAGES CXX)
+cmake_minimum_required(VERSION 3.17)
+project(
+  funi
+  VERSION 0.1.0
+  LANGUAGES CXX)
 
 if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+
 endif()
 
 # options - default is header only
```

Comment:
Is this comment correct? I see this as the default value for this option is 'ON', meaning ¿true?.

Reply:
good catch, updated!